### PR TITLE
Update files with PEP8 standards in folder dojo/tools #005

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/mozilla_observatory
dojo/tools/netsparker
dojo/tools/neuvector
dojo/tools/neuvector_compliance
dojo/tools/nexpose
dojo/tools/nikto
dojo/tools/nmap
dojo/tools/npm_audit
dojo/tools/nsp
dojo/tools/nuclei
dojo/tools/openscap
dojo/tools/openvas_csv
dojo/tools/ort
dojo/tools/ossindex_devaudit
dojo/tools/outpost24
dojo/tools/php_security_audit_v2
dojo/tools/php_symfony_security_check
dojo/tools/pip_audit
dojo/tools/pmd
dojo/tools/popeye
dojo/tools/pwn_sast
dojo/tools/qualys
dojo/tools/qualys_infrascan_webgui
dojo/tools/qualys_webapp
dojo/tools/retirejs
dojo/tools/risk_recon
dojo/tools/rubocop
dojo/tools/rusty_hog

### CODE REVIEW

File: dojo/tools/netsparker/parser.py [hunk 1]
```diff
@@ -20,26 +19,27 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8-sig'))
-        except:
+            data = json.loads(str(tree, "utf-8-sig"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/netsparker/parser.py:23]: Change to `Exception`

File: dojo/tools/netsparker/parser.py [hunk 2]
```diff
@@ -20,26 +19,27 @@ def get_description_for_scan_types(self, scan_type):
...
         for item in data["Vulnerabilities"]:
-
             title = item["Name"]
             findingdetail = html2text.html2text(item.get("Description", ""))
             if "Cwe" in item["Classification"]:
                 try:
-                    cwe = int(item["Classification"]["Cwe"].split(',')[0])
-                except:
+                    cwe = int(item["Classification"]["Cwe"].split(",")[0])
+                except BaseException:
```
Comment (Maffooch) [inline dojo/tools/netsparker/parser.py:36]: Change to `Exception`

File: dojo/tools/neuvector/parser.py [hunk 3]
```diff
@@ -22,82 +22,114 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/neuvector/parser.py:26]: Change to `Exception`

File: dojo/tools/neuvector/parser.py [hunk 4]
```diff
@@ -22,82 +22,114 @@ def parse_json(self, json_output):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/neuvector/parser.py:28]: Change to `Exception`

File: dojo/tools/neuvector_compliance/parser.py [hunk 5]
```diff
@@ -19,10 +19,10 @@ def parse_json(json_output):
     try:
         data = json_output.read()
         try:
-            tree = json.loads(str(data, 'utf-8'))
-        except:
+            tree = json.loads(str(data, "utf-8"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/neuvector_compliance/parser.py:23]: Change to `Exception`

File: dojo/tools/neuvector_compliance/parser.py [hunk 6]
```diff
@@ -19,10 +19,10 @@ def parse_json(json_output):
...
             tree = json.loads(data)
-    except:
+    except BaseException:
```
Comment (Maffooch) [inline dojo/tools/neuvector_compliance/parser.py:25]: Change to `Exception`

File: dojo/tools/npm_audit/parser.py [hunk 7]
```diff
@@ -29,22 +28,26 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/npm_audit/parser.py:32]: Change to `Exception`

File: dojo/tools/npm_audit/parser.py [hunk 8]
```diff
@@ -29,22 +28,26 @@ def parse_json(self, json_output):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/npm_audit/parser.py:34]: Change to `Exception`

File: dojo/tools/nsp/parser.py [hunk 9]
```diff
@@ -25,10 +24,10 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/nsp/parser.py:28]: Change to `Exception`

File: dojo/tools/nsp/parser.py [hunk 10]
```diff
@@ -25,10 +24,10 @@ def parse_json(self, json_output):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/nsp/parser.py:30]: Change to `Exception`

File: dojo/tools/ort/parser.py [hunk 11]
```diff
@@ -32,27 +31,32 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/ort/parser.py:35]: Change to `Exception`

File: dojo/tools/ort/parser.py [hunk 12]
```diff
@@ -32,27 +31,32 @@ def parse_json(self, json_output):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/ort/parser.py:37]: Change to `Exception`

File: dojo/tools/php_security_audit_v2/parser.py [hunk 13]
```diff
@@ -18,8 +17,8 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
+            data = json.loads(str(tree, "utf-8"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/php_security_audit_v2/parser.py:21]: Change to `Exception`

File: dojo/tools/php_symfony_security_check/parser.py [hunk 14]
```diff
@@ -24,10 +23,10 @@ def parse_json(self, json_file):
         try:
             data = json_file.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/php_symfony_security_check/parser.py:27]: Change to `Exception`

File: dojo/tools/php_symfony_security_check/parser.py [hunk 15]
```diff
@@ -24,10 +23,10 @@ def parse_json(self, json_file):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/php_symfony_security_check/parser.py:29]: Change to `Exception`

File: dojo/tools/qualys_webapp/parser.py [hunk 16]
```diff
@@ -73,24 +81,32 @@ def attach_unique_extras(endpoints, requests, responses, finding, date, qid, par
         port = ""  # Set port to empty string by default
         # Split the returned network address into host and
         try:  # If there is port number attached to host address
-            host, port = parsedUrl.netloc.split(':')
-        except:  # there's no port attached to address
+            host, port = parsedUrl.netloc.split(":")
+        except BaseException:  # there's no port attached to address
```
Comment (Maffooch) [inline dojo/tools/qualys_webapp/parser.py:85]: Change to `Exception`

File: dojo/tools/risk_recon/parser.py [hunk 17]
```diff
@@ -20,48 +19,75 @@ def get_findings(self, filename, test):
         if filename:
             tree = filename.read()
             try:
-                data = json.loads(str(tree, 'utf-8'))
-            except:
+                data = json.loads(str(tree, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/risk_recon/parser.py:23]: Change to `Exception`

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/risk_recon/parser.py (hunk 17 above)
Comment (ajtortolero) [reply dojo/tools/risk_recon/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/qualys_webapp/parser.py (hunk 16 above)
Comment (ajtortolero) [reply dojo/tools/qualys_webapp/parser.py:85]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/php_symfony_security_check/parser.py (hunk 15 above)
Comment (ajtortolero) [reply dojo/tools/php_symfony_security_check/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/php_symfony_security_check/parser.py (hunk 14 above)
Comment (ajtortolero) [reply dojo/tools/php_symfony_security_check/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/php_security_audit_v2/parser.py (hunk 13 above)
Comment (ajtortolero) [reply dojo/tools/php_security_audit_v2/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/ort/parser.py (hunk 12 above)
Comment (ajtortolero) [reply dojo/tools/ort/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/ort/parser.py (hunk 11 above)
Comment (ajtortolero) [reply dojo/tools/ort/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/nsp/parser.py (hunk 10 above)
Comment (ajtortolero) [reply dojo/tools/nsp/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/nsp/parser.py (hunk 9 above)
Comment (ajtortolero) [reply dojo/tools/nsp/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/npm_audit/parser.py (hunk 8 above)
Comment (ajtortolero) [reply dojo/tools/npm_audit/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/npm_audit/parser.py (hunk 7 above)
Comment (ajtortolero) [reply dojo/tools/npm_audit/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/neuvector_compliance/parser.py (hunk 6 above)
Comment (ajtortolero) [reply dojo/tools/neuvector_compliance/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/neuvector_compliance/parser.py (hunk 5 above)
Comment (ajtortolero) [reply dojo/tools/neuvector_compliance/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/neuvector/parser.py (hunk 4 above)
Comment (ajtortolero) [reply dojo/tools/neuvector/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/neuvector/parser.py (hunk 3 above)
Comment (ajtortolero) [reply dojo/tools/neuvector/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/netsparker/parser.py (hunk 2 above)
Comment (ajtortolero) [reply dojo/tools/netsparker/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

File: dojo/tools/netsparker/parser.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/tools/netsparker/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by devGregA — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Pull request merged**